95_windows/20_agent_installation.txt

```diff
@@ -2,6 +2,37 @@
 === Install Windows DSC agent
 
 The installation and upgrade processes are exactly the same.
+
+===== Supported version of Microsoft Windows
+
+The Rudder agent needs *PowerShell 4* or later, which is built-in on:
+
+ * Windows Server 2012 R2 and later
+
+PowerShell 4 may also be installed on the following platforms, following the this procedure: https://social.technet.microsoft.com/wiki/contents/articles/20623.step-by-step-upgrading-the-powershell-version-4-on-2008-r2.aspx 
+
+ * Windows Server 2008 R2
+ * Windows Server 2012
+
+===== Desktop version of Microsoft Windows
+
+There is no official support of desktop versions of Microsoft Windows. However, the agent can be installed on the following platform:
```

Comment:
There is no official support of /Rudder agent/ on desktop versions of Microsoft Windows.